qt/python/mantidqtinterfaces/test/Muon/PlottingView_test.py

```diff
@@ -1,420 +0,0 @@
-# Mantid Repository : https://github.com/mantidproject/mantid
```

Comment:
It looks like this code has been removed but maybe we should double check with @AnthonyLim23 that it hasn't been moved elsewhere?

Reply:
It should be there https://github.com/mantidproject/mantid/blob/main/qt/python/mantidqtinterfaces/test/Muon/PlottingView_test.py

Reply:
But `ElementalAnalysis/Plotting` (which is imports) is not a file or module in the project?

Reply:
this is from the first attempt of a separate plotting widget, which is also in Python. We needed to test the integration. As this is specific to muons it was decided that this was the best place for it.

Reply:
Looking into this, it seems like it was an unreliable test and was meant to have been deleted https://github.com/mantidproject/mantid/commit/fc0189ae6586d2e3f93d9bedfe6fd8ea18669059#diff-8ffab017c1c6a613568c272b90222401d627d251b1482b39feb82c78b9d9c4c3R40